wandb/wandb_torch.py

```diff
@@ -305,20 +305,20 @@ def _torch_hook_handle_is_valid(self, handle):
 
 
 class TorchGraph(wandb.data_types.Graph):
-    def __init__(self):
+    def __init__(self, full_hooks=True):
         super(TorchGraph, self).__init__("torch")
         # When we changed to register_full_backward_hook a regression test running fastai v1
         # started failing.  To maximize compatability we don't use full backward hooks
         # when we detect fastai v1 has been imported :(
-        self._should_use_full_hooks = True
+        self._should_use_full_hooks = full_hooks
```

Comment:
i prefer defaults to None, and then if none you would use True as long as some heuristics tell you it is ok. (like fastai v1).

If they say full_hooks=True, seems like we should use it.

but i dont feel strongly.

Reply:
Agreed, refactored to make it explicit.